dist2src/worker/config.py

```diff
@@ -23,19 +24,28 @@ def __init__(self):
 
         self._src_git_svc = None
         self._dist_git_svc = None
+        self._retries = Retry(
+            total=5,
+            backoff_factor=1,
+            status_forcelist=Retry.RETRY_AFTER_STATUS_CODES.union([404]),
+        )
 
     @property
     def src_git_svc(self) -> PagureService:
         if self._src_git_svc is None:
             self._src_git_svc = PagureService(
-                instance_url=f"https://{self.src_git_host}", token=self.src_git_token
+                instance_url=f"https://{self.src_git_host}",
+                token=self.src_git_token,
+                max_retries=self._retries,
```

Comment:
Now, when I see it, what about using `retry` or `retries` as an argument name for `PagureService`? (I can send a PR if we agree on it.)

Reply:
I've also thought about that name, but in the end I decided to stay consistent with the naming from [`requests.adapters.HTTPAdapter`](https://2.python-requests.org/en/master/api/#requests.adapters.HTTPAdapter). But I have no strong opinion on how to call this argument.

Reply:
Ok, that consistency makes sense. I am not against now.